src/components/Layout/navLinks.tsx

```diff
@@ -142,12 +142,12 @@ export const footerLinks: NavLink[] = [
                 href: '/blog',
             },
             {
-                title: 'Changelog',
-                href: 'https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/CHANGELOG.md',
+                title: 'Docs',
```

Comment:
Do we want to keep the duplicate Docs nav item that's outside of this Resources dropdown?

Reply:
Nope, thanks. Will fix.